src/api/github-repo/services/github-repo.js

```diff
@@ -120,6 +126,85 @@ class GithubManager {
     hasExpired(cacheEntry) {
         return cacheEntry.addedTime + CACHE_TTL < Date.now();
     }
+
+    async getInstallationToken() {
+        const now = Math.floor(Date.now() / 1000);
+
+        if (privateKey.startsWith('LS0t') || !privateKey.startsWith('-----BEGIN')) {
```

Comment:
We should make sure the private key is properly formatted in the environment variable.

Reply:
We already have the following secrets in kubernetes https://github.com/dev-launchers/k8s-infra/blob/main/tenants/base/strapiv4/secrets/integrations.yaml#L11-L13, so we just need to reference it in the env var. Ex:
```
        - name: GITHUB_APP_PRIVATE_KEY
          valueFrom:
            secretKeyRef:
              name: integrations
              key: github-strapi-invite-new-member-app-private-key
```